pom.xml

```diff
@@ -12,7 +12,7 @@
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
         <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
         <lwjgl.version>3.3.0-SNAPSHOT</lwjgl.version>
-        <swt.version>4.6.1</swt.version>
+        <swt.maven.version>3.105.3</swt.maven.version> <!-- contains Eclipse SWT 4.6.2.9 software -->
```

Comment:
Is there a reason why this property needs to be renamed? It should stay `swt.version`, IMO.

Reply:
I think keeping it as `swt.version` is misleading and incorrect, because it doesn't reflect the real version of the SWT library. Only in Maven Central does it have the 3.105.3 version in the coordinates, but the real SWT version is definitely 4.6.2.9 as indicated in the `version.txt` file inside the jar. So, 3.105.3 is just `swt.maven.version`, while 4.6.2.9 should be the real `swt.version` value but this real value doesn't exist in Maven Central!

I don't know why the 4.6.2.9 version number becomes 3.105.3 in Maven Central. Maybe the uploader didn't/doesn't care, due to some technical limitation in the conversion tool from p2 repo to Maven repo. But I do care, hence the 3.105.3 number is `swt.maven.version` and not `swt.version`, an explicit distinction.